Sources/TSCUtility/PkgConfig.swift

```diff
@@ -14,16 +14,16 @@ import Foundation
 public enum PkgConfigError: Swift.Error, CustomStringConvertible {
     case couldNotFindConfigFile(name: String)
     case parsingError(String)
-    case nonWhitelistedFlags(String)
+    case nonAllowlistedFlags(String)
 
     public var description: String {
         switch self {
         case .couldNotFindConfigFile(let name):
             return "couldn't find pc file for \(name)"
         case .parsingError(let error):
             return "parsing error(s): \(error)"
-        case .nonWhitelistedFlags(let flags):
-            return "non whitelisted flag(s): \(flags)"
+        case .nonAllowlistedFlags(let flags):
+            return "not allowed flag(s): \(flags)"
```

Comment:
Maybe a nitpick, but would "prohibited flags" make more sense here?

Reply:
No, that's good. I was thinking "nonAllowlisted" sounds a bit strange, but didn't come up with a better alternative.